pcdsdevices/interface.py

```diff
@@ -1304,6 +1328,17 @@ def state(self):
                     closest = diff
         return state
 
+    @property
+    def sync_needed(self) -> bool:
```

Comment:
One last tiny tiny nitpick: this is a property that touches the filesystem, usually I prefer things that may take time to be function calls. Usually we expect attribute access to be fast, so if they are not fast we can accidentally write code with performance issues.

Reply:
A good point.  I'll do that